src/aap_eda/api/views/activation.py

```diff
@@ -87,48 +68,41 @@ class ActivationViewSet(
     @extend_schema(
         request=serializers.ActivationCreateSerializer,
         responses={
-            status.HTTP_201_CREATED: serializers.ActivationReadSerializer
+            status.HTTP_201_CREATED: serializers.ActivationReadSerializer,
+            status.HTTP_400_BAD_REQUEST: OpenApiResponse(
+                description="Invalid data to create activation."
+            ),
         },
     )
     def create(self, request):
-        serializer = serializers.ActivationCreateSerializer(data=request.data)
-        serializer.is_valid(raise_exception=True)
-
-        try:
-            response = serializer.create(
-                serializer.validated_data, request.user
+        context = {"request": request}
+        serializer = serializers.ActivationCreateSerializer(
+            data=request.data, context=context
+        )
+        valid = serializer.is_valid()
```

Comment:
@hsong-rh Can we just have it raise the exception wouldn't that be converted to a bad request?

Reply:
The reason we removed `raise_exception=True` from `is_valid` is because we want to rename `non_field_errors`